system/lib/fetch/wasmfs.cpp

```diff
@@ -0,0 +1,55 @@
+// // Copyright 2021 The Emscripten Authors.  All rights reserved.
+// // Emscripten is available under two separate licenses, the MIT license and the
+// // University of Illinois/NCSA Open Source License.  Both these licenses can be
+// // found in the LICENSE file.
+
+#include <stdlib.h>
+#define __NEED_struct_iovec
+#include <emscripten/emscripten.h>
+#include <wasi/api.h>
+
+extern "C" {
+
+  __wasi_errno_t __wasi_fd_write( __wasi_fd_t fd, const __wasi_ciovec_t *iovs, size_t iovs_len, __wasi_size_t *nwritten) {
```

Comment:
This line looks longer than 80 characters, so it would be nice to run `git clang-format` to standardize the formatting of the C/C++ source. @kripken or @sbc100, is that standard practice in this repo? Perhaps we should have CI check formatting like we do in Binaryen.

On the other hand, maybe clang-format does something terrible to `EM_ASM` macros :/

Reply:
I can use clang-format if that's appropriate for sure!